frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss

```diff
@@ -36,10 +33,16 @@
             border-style: solid;
             border-width: var(--lemon-button-border-width);
             border-radius: var(--radius);
-            transition: var(--transition);
 
             .LemonButton__icon {
-                opacity: 0.5;
+                color: var(--trace-3000);
+                transition: none;
+
+                > :not(.LemonIcon) {
+                    // Elements that serve as an icon, but aren't an icon are slightly muted -
+                    // not so much that they look disabled though
+                    opacity: 0.75;
```

Comment:
This is generalized, so applies not just to ProfilePicture, but also e.g. Lettermark